docs/hub/models-gguf.md

```diff
@@ -0,0 +1,43 @@
+# GGUF
+
+Hugging Face Hub supports [GGUF format](https://github.com/ggerganov/ggml/blob/master/docs/gguf.md), a binary format that is optimized for quick loading and saving of models, making it highly efficient for inference purposes. Models initially developed in frameworks like PyTorch can be converted to GGUF format. GGUF is designed for use with GGML and other executors. GGUF was developed by [@ggerganov](https://huggingface.co/ggerganov) who is also the developer of [llama.cpp](https://github.com/ggerganov/llama.cpp), a popular C/C++ LLM inference framework.
+
+<div class="flex justify-center w-full">
+<img style="max-width: 50em;" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/gguf-spec.png"/>
+</div>
+
+## Finding GGUF files
+
+You can browse all models with GGUF weights thanks to GGUF tag: [hf.co/models?library=gguf](https://huggingface.co/models?library=gguf).
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/gguf-filter-light.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/gguf-filter-dark.png"/>
+</div>
+
+For example, you can check out [TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF](https://huggingface.co/TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF) for seeing GGUF weights in action. 
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/gguf-repo-light.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/gguf-repo-dark.png"/>
+</div>
+
+> [!tip]  
+> More features regarding GGUF support on Hugging Face Hub are coming soon 🚀
```

Comment:
Yes! would include how to directly download and run a gguf with llama.cpp here: https://x.com/ggerganov/status/1758136382978183236?s=20

Reply:
thanks so much for the link. I was about to ask where would I find the example.

Handled in ca3fc6134bf0d52039d0acc3476802b499bd8c31